types/react-modal-video/vendors.d.ts

```diff
@@ -0,0 +1,54 @@
+// this interfaces here are somewhat volatile, do not close them,
+// their value is documental, they only resolve to an url query param.
+// like so: https://www.vendor.com?[key]=[value]&...
+// None of this apis is versionated. Update or change any field should be OK.
+
+export interface VimeoApi {
+    api?: boolean;
+    autopause?: boolean;
+    autoplay?: boolean;
+    byline?: boolean;
+    callback?: string;
+    color?: string;
+    height?: number;
+    loop?: boolean;
+    maxheight?: number;
+    maxwidth?: number;
+    player_id?: string;
+    portrait?: boolean;
+    title?: boolean;
+    width?: boolean;
+    xhtml?: true;
+    [K: string]: any;
```

Comment:
```suggestion
    [key: string]: any;
```
convention for key/value?